### DISCUSSION TITLE
Do the students and colleague lecturers have access to copilot through an academic organization ?

### DISCUSSION THREAD

**Discussion by yannclaveau** [community/community#60332] [GitHub Education]
Hi, 

Do the students and colleague lecturers have access to copilot through an academic organization ? Or do they have to subscribe to a student / teacher account on their own ?

I'd like to create a classroom so that the students can use copilot (ang github). I would like to avoid losing time in classroom by asking for individual access to copilot for the students, but we can't afford the business plan.

On the other hand, we are several lecturers and we would like to all have access to copilot. All of us have to subscribe on our own ?

Thanks !
Yann

**Comment by samus-aran** [comment]
Hi @yannclaveau 

Yes you would need to apply individually either as a student or a teacher. Each user needs to be verified to receive the benefits. 

Hope that helps 👍
